plugins/plugin-chart-echarts/src/Funnel/transformProps.ts

```diff
@@ -181,11 +181,12 @@ export default function transformProps(
           fontWeight: 'bold',
         },
       },
+      // @ts-ignore
```

Comment:
There seems to be a minor regression in the `FunnelSeriesOption` interface (will look into fixing this upstream later)